aries_cloudagent/protocols/issue_credential/v2_0/models/detail/vc_di.py

```diff
@@ -0,0 +1,159 @@
+"""VCDI-specific credential exchange information with non-secrets storage."""
+
+from typing import Any, Mapping, Sequence
+
+from marshmallow import EXCLUDE, fields
+
+from ......core.profile import ProfileSession
+from ......messaging.models.base_record import BaseRecord, BaseRecordSchema
+from ......messaging.valid import (
+    INDY_CRED_REV_ID_EXAMPLE,
+    INDY_CRED_REV_ID_VALIDATE,
+    INDY_REV_REG_ID_EXAMPLE,
+    INDY_REV_REG_ID_VALIDATE,
+    UUID4_EXAMPLE,
+)
+from .. import UNENCRYPTED_TAGS
+
+
+class V20CredExRecordVCDI(BaseRecord):
```

Comment:
I don't think we need a special VCDI credential exchange class, we should just be able to use the "old" `V20CredExRecord` class.  VCDI is just an attachment format, and the `V20CredExRecord` can already handle multiple types of attachments.  Having a separate type of exchange record for VCDI complicates things IMHO.  If there is anything VCDI-specific we should be able to include it in the attachment.